coding-concepts.js

```diff
@@ -16,44 +16,51 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
 
 // a) Your answer:
+// I THINK THE ANSWER IS TEN
 // b) Verify and explain:
-
+// AFTER THE CONSOLE LOG.... THE ANSWER IS 10.... I NOTICE THE .length PROPERTY ------L E A R N  2 0 2 2---- I COUNTED 10 SPACES... MY GUESS WAS CORRECT
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
 // a) Your answer:
+// I THINK THE ANSWER IS "O"
 // b) Verify and explain:
+// AFTER THE LOG.... THE ANSWER IS "O" ....[4] IS THE INDEX VALUE --------H E L L O--------- I COUNTED 4 SPACES (STARTING FROM 0)... MY GUESS WAS CORRECT 
 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
 // a) Your answer:
+// I THINK THE ANSWER IS Ruby
 // b) Verify and explain:
+// AFTER THE LOG.... THE ANSWER IS Ruby ...."Javascript IS ZERO... "Ruby" IS ONE... "Rubyß" IS THE FIRST INDEX FOR THIS LIST OF STRINGS
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
+// a) Your answer: I THINK THE ANSWER IS "SATURDAY" "SUNDAY"
 // b) Verify and explain:
-
+// AFTER THE LOG.... THE ANSWER IS "TypeError: weekendDays.toUpperCase is not a function"... I AM GOING TO FURTHER RESEARCH HOW TO USE THE UPPER CASE FOR THIS SITUATION
```

Comment:
.toUpperCase only works on strings and not on arrays.  A way achieve the result you mentioned on line 54 would be to iterate over the array and run .toUpperCase on the individual strings.